multiversx_sdk/core/transactions_outcome_parsers/transaction_events_parser.py

```diff
@@ -26,14 +24,11 @@ def parse_events(self, events: List[TransactionEvent]) -> List[SimpleNamespace]:
     def parse_event(self, event: TransactionEvent) -> SimpleNamespace:
         first_topic = event.topics[0].decode() if len(event.topics) else ""
         abi_identifier = first_topic if first_topic and self.first_topic_as_identifier else event.identifier
-
         topics = event.topics
-
         if self.first_topic_as_identifier:
             topics = topics[1:]
-
         return self.abi.decode_event(
             event_name=abi_identifier,
             topics=topics,
-            data_items=event.data_items,
+            data_items=event.additional_data,
```

Comment:
Here, we still use the name `data_items`. What should we do (in a future PR), and also on the specs?

Reply:
as discussed, will be renamed in a future PR.